Disable TestRoutingTableProviderPeriodicRefresh #1250

kaisun2000 opened the pull request:
### Issues

- [x] My PR addresses the following Helix issues and references them in the PR description:

Disable #1250 

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

TestRoutingTableProviderPeriodicRefresh use sleep() waiting for timers
to trigger. This is the source of frequent failure. This test needs a
re-write. Current state is hard to fix.

### Tests

- [ ] The following tests are written for this issue:

(List the names of added unit/integration tests)

- [x] The following is the result of the "mvn test" command on the appropriate module:

github running
### Documentation (Optional)

- In case of new functionality, my PR adds documentation in the following wiki page:

(Link the GitHub wiki you added)

### Commits

- My commits all reference appropriate Apache Helix GitHub issues in their subject lines. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Code Quality

- My diff has been formatted using helix-style.xml 
(helix-style-intellij.xml if IntelliJ IDE is used)

jiajunwang commented:
I'm pretty sure this is not fixing #1250. Please don't describe the PR in this confusing way.

Also, please mention this issue number in the code comment so we have a tight connection between TODOs and issues.

kaisun2000 commented:
> I'm pretty sure this is not fixing #1250. Please don't describe the PR in this confusing way.
> 
> Also, please mention this issue number in the code comment so we have a tight connection between TODOs and issues.
Good point, changed the description to disable instead of fix. Also added the tracking issue number as todo.

jiajunwang commented:
With my recent change, I think this change is no longer needed. Although it is still not idea test logic, but better than disabling it, IMHO.
